espnet2/tasks/enh.py

```diff
@@ -328,6 +328,28 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             help="The set of all possible categories in the dataset. Used to add the "
             "category information to each sample",
         )
+        group.add_argument(
+            "--speech_segment",
+            type=int_or_none,
+            default=None,
+            help="Truncate the audios to the specified length if not None",
```

Comment:
Truncate the audio to the specified length (If I understand correctly, the length is in sampling points?)

Reply:
Yes it's number of samples. Have updated the help-string.